inc/namespace.php

```diff
@@ -438,7 +438,11 @@ function get_valid_dimensions( $size, int $max_width = 0, int $max_height = 0 )
 	return [ $width, $height, $crop ];
 }
 
-function dynamic_downsize( $downsize, int $attachment_id, $size ) {
+function dynamic_downsize( $downsize, $attachment_id, $size ) {
+	if ( ! $attachment_id ) {
+		return $downsize;
+	}
+	
```

Comment:
Isn't it enough to remove the type declaration? If I understood correctly, this is where the error is thrown (due to strict types).

```suggestion
```

If the attachment ID is empty, there is no attachment, and so the function bails when checking for a valid provider. There will not be one.